code/game/machinery/computer/records/security.dm

```diff
@@ -228,7 +228,7 @@
 	var/datum/crime/citation/new_citation = new(name = input_name, details = input_details, author = usr, fine = params["fine"])
 
 	target.citations += new_citation
-	new_citation.alert_owner(user, src, target.name, "You have been issued a [params["fine"]]cr citation for [input_name]. Fines are payable at Security.")
+	new_citation.alert_owner(user, src, target.name, "You have been issued a [params["fine"]]cr citation for [sanitize(input_name)]. Fines are payable at Security.")
```

Comment:
Shouldn't we sanitize this up earlier when we actually actually set the var? `input_details = trim(params["details"], MAX_MESSAGE_LEN)`?

Reply:
I was wondering the same thing, but this is the only portion to my knowledge that treats the input as actual html. If done on the input, it shows the encoded version in the security console GUI. And I'm not sure how to fix that.

Reply:
are you sure `strip_html_full()` doesn't work isntead of just `sanitize()`?

cc: @jlsnow301 any ideas

Reply:
I do think that'd work fine, imo it'd be better to try and preserve the input 

I'm not sure what the standard is, if you're going to have to decode html every time you send data to a ui, does it make sense still to store an encoded version?

Reply:
it would make sense to just strip it of any excess entirely, storing it or not. we don't want a vector for user injectable scripts. look at what papercode does for example

Reply:
strip_html_full does also call sanitize, so any unmatched <> will be broken. I've adjusted to the suggestions just in case that's an acceptable tradeoff, which should fix *another*, much longer dangerous input, with missing/wanted poster descriptions
![image](https://github.com/tgstation/tgstation/assets/25628932/802e88c2-1675-40af-b49f-53979669fc70)

Reply:
we can audit more later but i think it's better to at least get this one through the door sooner rather than later